app/microsoft/helpers.py

```diff
@@ -0,0 +1,45 @@
+import msal
+import logging
+
+logger = logging.getLogger(__name__)
+
+# Define constants for making requests to Graph API.
+BASE_URL = "https://graph.microsoft.com/v1.0/"
+
+HTTP_HEADERS = {
+    "Accept": "application/json",
+    "Content-Type": "application/json",
+}
+
```

Comment:
By convention documentation for Python functions and classes is written as follows:

```python
def create_client(client_id, authority_url, client_secret):
    """Authenticate our app with Azure Active Directory."""
    # ... code goes here 
```

For future code changes please follow this convention.